packages/contractkit/src/wrappers/LockedGold.ts

```diff
@@ -134,10 +108,11 @@ export class LockedGoldWrapper extends BaseWrapper<LockedGold> {
   }
 
   async getAccountSummary(account: string): Promise<AccountSummary> {
+    const accounts = await this.kit.contracts.getAccounts()
     const nonvoting = await this.getAccountNonvotingLockedGold(account)
     const total = await this.getAccountTotalLockedGold(account)
-    const voter = await this.getVoterFromAccount(account)
-    const validator = await this.getValidatorFromAccount(account)
+    const voter = await accounts.getVoteSignerFromAccount(account)
```

Comment:
I would remove these from the account summary, and possibly rename this function

Reply:
Removed them from the summary, but not sure what a better name would be